src/util/serialize.h

```diff
@@ -426,13 +498,19 @@ MAKE_STREAM_WRITE_FXN(s32,   S32,      4);
 MAKE_STREAM_WRITE_FXN(s64,   S64,      8);
 MAKE_STREAM_WRITE_FXN(f32,   F1000,    4);
 MAKE_STREAM_WRITE_FXN(f32,   F32,      4);
+MAKE_STREAM_WRITE_FXN(double,   F32,      4);
```

Comment:
I think it'd be better if the caller explicitly casted to float if they want to use writeF32 with a double.

Reply:
I removed overloaded support for serializing F64 using the F32 serialization.